core/optic/shared/src/main/scala/com/useoptic/contexts/requests/projections/AllEndpointsProjection.scala

```diff
@@ -0,0 +1,31 @@
+package com.useoptic.contexts.requests.projections
+
+import com.useoptic.contexts.requests.Commands.PathComponentId
+import com.useoptic.contexts.rfc.RfcState
+
+import scala.scalajs.js.annotation.JSExportAll
+
+object AllEndpointsProjection {
```

Comment:
A "Projection" must be built from the event stream. I would only name things built from the event stream *Projection